assets/scripts/helperFunctions.js

```diff
@@ -59,7 +59,7 @@ const savePackingListToStorage = (listName, list) => {
   const currentPackingList = getPackingListFromStorage();
   currentPackingList.push({
     name: listName,
-    items: list
+    items: list,
   });
```

Comment:
why?

Reply:
uh. I had added something and then removed it. so there was an extra comma.